eth/vm/forks/frontier/state.py

```diff
@@ -142,21 +142,28 @@ def build_computation(self,
 
         return computation
 
-    def finalize_computation(self,
-                             transaction: SignedTransactionAPI,
-                             computation: ComputationAPI) -> ComputationAPI:
-        transaction_context = self.vm_state.get_transaction_context(transaction)
+    def calculate_gas_refund(self,
```

Comment:
Looks like this could be a classmethod. That helps show that it doesn't depend on anything besides the passed-in variables.